<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="assetGroup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Add an Asset!</string>
   </property>
   <widget class="QLabel" name="assetNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="assetCostLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="assetCategoryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="assetDescriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="assetNameBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="assetCostBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QComboBox" name="assetCategoryCombo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="assetDescriptionText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addAsset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Asset</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="categoryGroup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>218</width>
     <height>97</height>
    </rect>
   </property>
   <property name="title">
    <string>Add a Category!</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="categoryNameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="categoryNameBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="addCategory">
      <property name="text">
       <string>Add Category</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
